packages/build/tests/plugins/fixtures/functions_list/plugin.js

```diff
@@ -8,6 +8,6 @@ module.exports = {
   }) {
     const functions = await list()
 
-    console.log(functions)
+    console.log(JSON.stringify(functions, null, 2))
```

Comment:
Stringifying the functions content to reproduce the same output for both Node versions.